frontend/wallet/src/render/components/ToggleButton.re

```diff
@@ -0,0 +1,57 @@
+module Styles = {
+  open Css;
+
+  let container = style([display(`flex), height(`rem(2.))]);
+
+  let button = active_ =>
```

Comment:
up to u - maybe rename this isSelected to avoid the _?

Reply:
I think i'm just gonna leave it.